sources/Outputs/src/ParGenerator.cpp

```diff
@@ -223,7 +223,7 @@ void ParGenerator::updateSignalNGenerator(boost::shared_ptr<parameters::Paramete
   }
 
   set->addReference(helper::buildReference("generator_PRef0Pu", "targetP_pu", "DOUBLE"));
-  if (set->getId() == constants::remoteVControlParId) {
+  if (set->getId() == constants::remoteVControlParId || set->getId() == constants::remoteVControlParId + "_Nuc") {
```

Comment:
Wouldn't it be more rigorous to create a new constant (named for e.g. `remoteVControlNucParId`)?

Reply:
done